app/views/well_known/host_meta/show.xml.ruby

```diff
@@ -5,7 +5,7 @@ doc << Ox::Element.new('XRD').tap do |xrd|
 
   xrd << Ox::Element.new('Link').tap do |link|
     link['rel']      = 'lrdd'
-    link['type']     = 'application/xrd+xml'
+    link['type']     = 'application/jrd+json'
```

Comment:
That would mean breaking compatibility with implementations which only handle XRD for LLRD (I don't know if there is any). Maybe we could advertise both.

Reply:
I don't think this is a practical concern.